style.css

```diff
@@ -0,0 +1,82 @@
+/* Box sizing rules */
```

Comment:
If this is a 'normalize' file please call it like that since we're going to have our own style.

Let's not clog the main CSS file with 'normalize' or another styling snippet.